Main/src/Coffee/FindMin.java

```diff
@@ -0,0 +1,30 @@
+package Coffee;
+
+public class FindMin {
+
+
+
+    public static float min(float a, float b, float c){
+        if (a<b) {
```

Comment:
@GayaneGulanyan for calculating min or other mathematical functions you can use Math library

Reply:
done